e2e/configurationEditor.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from '@grafana/plugin-e2e';
+import { BitmovinDataSourceOptions } from '../src/types/grafanaTypes';
+
+test('"Save & test" should be successful when configuration is valid', async ({
```

Comment:
can we split it and create 
- top level `describe('configurationEdtor')`
  - under to have `it('should save and test valid configuration')` (your scenario)
  - and another unsuccessfull scenario `it('should not save invalid configuration')`, where we use use `test-api-key` string and try to save it (do not mock http responses, let backend tell us licenses cant be fetched)

Reply:
I added the second test but there is no such thing as `describe` in playwright tests.